package.json

```diff
@@ -45,6 +45,7 @@
     "@babel/core": "^7.5.5",
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
     "@babel/preset-env": "^7.5.5",
+    "@rollup/plugin-replace": "^2.2.1",
```

Comment:
shouldn't this come in addition to a change to the rollup config?

or was this just an accidental inclusion from another PR?

Reply:
Yeah good catch - this is for `process.env.BABEL_ENV`. I made the change to the rollup config but lost it in my frantic switching between branches. Will reinstate.